sample/core.py

```diff
@@ -139,7 +139,64 @@ def get_redirects(self, rdlimit="max", rdprop="pageid|title"):
         self.payload.pop('rdcontinue', None)
         self.payload['prop'] = None
         return rd_list
-    
+
+
+    def get_categoryinfo(self):
+        prop = 'categoryinfo'
+        self.payload['prop'] = prop
+
+        res = requests.get(self.base_url, params=self.payload, headers=self.headers).json()
+
+        ci_list = {}
+        for page_id, page_content in res['query']['pages'].items():
+            if prop not in page_content:
+                continue
+            ci_list[page_id] = page_content[prop]
+
+        while 'continue' in res:
+            self.payload['cicontinue'] = res['continue']['cicontinue']
+            res = requests.get(self.base_url, params=self.payload, headers=self.headers).json()
+            for page_id, page_content in res['query']['pages'].items():
+                if prop not in page_content:
+                    continue
+                if page_id not in ci_list:
+                    ci_list[page_id] = []
+                ci_list[page_id] = page_content[prop]
+
+        self.payload.pop('cicontinue', None)
+        self.payload['prop'] = None
+        return ci_list
+
+    def get_duplicatefiles(self, dflimit="max", dfdir="ascending", dflocalonly=None):
+        prop = 'duplicatefiles'
+        self.payload['prop'] = prop
+        self.payload['dflimit'] = dflimit
+        self.payload['dfdir'] = dfdir
+        if dflocalonly is not None:
```

Comment:
`True` would be more appropriate default value for `dflocalonly`. That way people can figure out the default from the documentation itself and also that they need to set it to a value like `False`.